src/filters.js

```diff
@@ -12,8 +12,17 @@ const ALLOWED_ATTRS_BY_TAG_FOR_HEAD_MARKUP = {
   script: ['type'],
 };
 
+// Only load window from jsdom when it does not exist. In a node env, we expect
+// `window` to be undefined all the time but we are using the "jsdom"
+// environment in Jest, which already provides `window`. We don't want to load
+// jsdom again when that is the case.
+if (typeof window === 'undefined') {
+  // eslint-disable-next-line global-require
+  const { JSDOM } = require('jsdom');
+  global.window = new JSDOM('').window;
+}
```

Comment:
@bobsilverberg WDYT? (given you had to fix the same issue in addons-frontend)

Reply:
It looks reasonable and I think we can take the same approach with addons-frontend. I hadn't thought about doing an conditional import with `require`, but it seems to work.